dev/localize/en-ios.json

```diff
@@ -14254,6 +14254,10 @@
     "key": "ReportPeer.AlertSuccess",
     "value": "Thank you!\nYour report will be reviewed by our team."
   },
+  "ReportPeer.AlertFailure": {
```

Comment:
No need to edit these files, they are redundant and will be removed from the repo soon.

Reply:
@Ajaxy, ok, there are non-localized fail cases as before.